File: JS/Sequencial/EX4_A.js
/*
4.Faca um programa que receba o salario de um funcionario, calcule e mostre o novo salario, sabendo-se que este sofreu um aumento de 25%.
*/

confirm("Verificação do aumento de 25% do salário")

var salario = prompt("Digite o salario atual ")

salario = parseFloat(salario)
var reajuste = salario + (salario * 25/100)
reajuste = parseFloat(reajuste)


console.log("O reajuste será de: "+reajuste.toLocaleString('pt-BR', { style: 'currency', currency: 'BRL' }))

File: python/Sequencial/EX4_A.py
#4.Faca um programa que receba o salario de um funcionario, calcule e mostre o novo salario, sabendo-se que este sofreu um aumento de 25%.

print("********************SEQUENCIAL*************************")
print("********************Programa*4*************************")
print("*********Cálculo do aumento salarial de 25%  **********")
print("*******************************************************")

salario = float(input("Veja o aumento de 25% no seu salário: "))
novo_salario = salario + (salario * 25/100);

print('Salario anterior: R$ {: .2f} | Salario Atual: R$ {: .2f}.'.format(salario, novo_salario))
